File: Java/1067.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        
        long a = sc.nextLong();
        
        if(a < 0) System.out.printf("minus\n");
        else System.out.printf("plus\n");
        
        if(a%2==0) System.out.printf("even");
        else System.out.printf("odd");
    }
}

File: C++/1067.cpp
#include <iostream>

using namespace std;

int main(){
    int a;
    cin >> a;
    
    if(a < 0) cout << "minus\n";
    else cout << "plus\n";
    
    if(a%2==0) cout << "even";
    else cout << "odd";
}